tests/core/creators/detached.js

```diff
@@ -0,0 +1,150 @@
+/* bender-tags: editor, 4.17.0, feature, 4461*/
+/* bender-ckeditor-plugins: wysiwygarea */
+
+( function() {
+	'use strict';
+	var tests = {
+		'test delay editor creation if target element is detached': function() {
+			var editorElement = CKEDITOR.document.getById( 'editor1' ).remove();
+
+			var editor = CKEDITOR.replace( editorElement, {
+				delayIfDetached: true
+			} );
+
+			assert.isNull( editor );
+		},
+
+		'test editor is created immediately on not detached element even with delay config': function() {
+			var editorElement = CKEDITOR.document.getById( 'editor5' );
```

Comment:
Can we change the fixtures order to follow the tests order (so that we had `editor2` here instead of `editor5`)?